tools/build-archive.sh

```
@@ -0,0 +1,48 @@
+
+# 一時ステージングディレクトリで開発専用ファイルを除外してから固める
+STAGE="$(mktemp -d)"
+trap 'rm -rf "${STAGE}"' EXIT
+
+cp -R "${PLUGIN_DIR}/." "${STAGE}/"
+
+# 開発環境専用ファイルは配布アーカイブから除外する
+rm -rf "${STAGE}/tools"
+
+cd "${STAGE}"
+tar czf "${OUTPUT}" *
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)